### PR TITLE
Feature split panes sub1

### PR DESCRIPTION
**Pull request by CCole**
Implemented basic functioning Directive

### CODE REVIEW

File: public/js/splitjs/splitjsDir.directive.js [hunk 1]
```diff
@@ -2,19 +2,19 @@
 
 module.exports =
 angular
-.module('diSplitJs.directives', [])
-.directive('split-area'), function(){
```
Comment (CCole) [inline public/js/splitjs/splitjsDir.directive.js]: @JuanCaicedo here's where I went wrong. Notice that the parens executing the function are misplaced

**Comment by joemccann** [comment]
sorry is this PR no longer valid?  Why was it closed?

**Comment by JuanCaicedo** [comment]
@CCole are actually working on a branch on his fork before we submit it here. We didn't want to make extra noise until it's ready :)

**Comment by CCole** [comment]
sorry about that @joemccann I def didn't mean to make the pull request to your master branch yet.

**Comment by joemccann** [comment]
Ok cook

**Comment by joemccann** [comment]
Cool*

**Pull request closed without merging**